apps/src/studio/studio.js

```diff
@@ -5604,7 +5605,7 @@ Studio.fixSpriteLocation = function() {
  */
 Studio.setSprite = function(opts) {
   if (typeof opts.value !== 'string') {
-    throw new TypeError('Incorrect parameter: ' + opts.value);
```

Comment:
do we rely on these errors in any way? (i.e., do we display them to the student in the console, or otherwise rely on this function throwing rather than no-oping)

Reply:
That's a great point. I'm not sure. I'll go ahead and split this into a different PR so we can get the other patches in while I investigate.